shared/EShopOnAbp.Keycloak.DbMigrator/KeycloakDataSeeder.cs

```diff
@@ -329,6 +330,16 @@ private async Task CreateSwaggerClientAsync()
             };
 
             await _keycloakClient.CreateClientAsync(_keycloakOptions.RealmName, swaggerClient);
+
+            // Add Scopes For ClientId
+            await AddOptionalClientScopesAsync(
+                clientId,
+                new List<string>
+                {
+                    "AdministrationService", "IdentityService", "BasketService", "CatalogService",
+                    "OrderingService", "PaymentService", "CmskitService"
+                }
+            );
```

Comment:
@gterdem 
Add Scopes For ClientId (SwaggerClient & WebGateway_Swagger)

Reply:
This is fixed with https://github.com/abpframework/eShopOnAbp/pull/238/commits/704fd816e35d3d7d4c22dc4d0e13fa15eae4acec